tinker/templates/events/form.html

```diff
@@ -339,19 +339,29 @@ <h3 class="subtitle">{{ field.label }}</h3>
             };
 
 
-            function offCampusLocation() {
+            function offCampusLocation(clear_values) {
+                if (clear_values){ //Clears other options if the user switches in between them
+                    $("#off_campus_location").val('');
+                    $("#on_campus_location").val('none');
+                    $("#other_on_campus").val('');
+                }
                 var location = $("#location")[0];
                 var val = location.options[location.selectedIndex].innerHTML;
                 if (val == "Off Campus") {
                     $("#off_location_wrap").show();
                     $("#on_campus_wrap").hide();
                     $("#other_wrap").hide();
                 }
-                else {
+                else if(val == "Other On Campus"){
                     $("#off_location_wrap").hide();
-                    $("#on_campus_wrap").show();
+                    $("#on_campus_wrap").hide();
                     $("#other_wrap").show();
```

Comment:
This is super picky, but can you order these hide/shows for all of these in the same order they appear in the dropdown? (on campas, other on campus, off campus).

Reply:
Did this.